eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -47,4 +47,10 @@
     <!-- Transitive dependency of System.Security.Cryptography.Pkcs.5.0.*. -->
     <UsagePattern IdentityGlob="System.Formats.Asn1/6.0.0" />
   </IgnorePatterns>
+
+  <!-- Transitive dependencies, suppressed as per https://github.com/dotnet/aspnetcore/pull/57887#issuecomment-2362241518-->
```

Comment:
@dotnet/source-build-contrib 

Do we need a tracking issue for this? Do we know why/how the behavior changed?

Reply:
@ellahathaway - Can you respond here since you had investigated this in https://github.com/dotnet/aspnetcore/pull/57887#issuecomment-2362241518?

Reply:
> Do we need a tracking issue for this?
I don't think that a tracking issue is necessary here. On the condition that these prebuilts do cause failures in the VMR, source-build will see those failures in our pipelines, file an issue in the source-build repo, and I can follow up there. Unless that happens, I don't see a need to write up a tracking issue until there is some (if any) action to take.

> Do we know why/how the behavior changed?
Unfortunately, no. The binlogs were not helpful in diagnosing the issue, but we found packages (that were already listed in the exclusion baseline) in the prebuilt report that are known parents of these dependencies. Based on that, we made an educated guess that these prebuilts likely surfaced due to changes in how nuget handles transitive dependencies.